pipeline_dp/combiners.py

```diff
@@ -208,3 +187,29 @@ def compute_metrics(
         for combiner, acc in zip(self._combiners, accumulator):
             metrics.append(combiner.compute_metrics(acc))
         return (privacy_id_count, metrics)
+
+
+def create_compound_combiners(
+        aggregate_params: pipeline_dp.AggregateParams,
+        budget_accountant: budget_accounting.BudgetAccountant
+) -> CompoundCombiner:
+    combiners = []
+    mechanism_type = aggregate_params.noise_kind.convert_to_mechanism_type()
+
+    if pipeline_dp.Metrics.COUNT in aggregate_params.metrics:
```

Comment:
Are all metrics in aggregate_params.metrics expected to have a combiner?

If yes, it might be safer to iterate through the list, and then throw an exception if there's a metric for which we don't support a combiner. 

for metric in aggregate_params.metric:
  if metric == pipeline_dp.Metrics.COUNT:
    ..
  elseif metric = pipeline_dp.Metrics.SUM:
    ..

  else:
     throw Error ("Unsupporter metric:" + metric)


WDYT?

Reply:
Actually it's not 1:1 correspondence, 1 combiner can have more than 1 metrics to compute. E.g. MeanCombiner (which is currently WIP), this Combiner along the way computes Count and Sum. So with adding MeanCombiner this function needs to be significantly extended.

Reply:
Thanks for the explanation! I'm still a bit unsure how the mean combiner will look here, but maybe we can defer this until we have the mean combiner. Thanks!